packages/design-system/src/components/Spinner/Spinner.tsx

```diff
@@ -40,12 +40,16 @@ export const Spinner: React.FunctionComponent<SpinnerProps> = (props: SpinnerPro
     props.className
   );
 
-  return <span className={className} aria-valuetext={props['aria-valuetext']} role={props.role} />;
+  return (
+    <span className={className} role={props.role}>
+      <span className="ds-u-visibility--screen-reader">{props['aria-valuetext']}</span>
+    </span>
+  );
 };
 
 Spinner.defaultProps = {
   'aria-valuetext': 'Loading',
-  role: 'progressbar',
+  role: 'status',
```

Comment:
Do we want role to even be a prop? What other role would they want? If they wanted `progressbar` again, they wouldn't have the ability to add the `aria-label` or whatever was needed to make it accessible, right?